XiEditor/EditViewContainer.swift

```diff
@@ -0,0 +1,22 @@
+// Copyright 2016 Google Inc. All rights reserved.
```

Comment:
It's 2017 now :)

Not urgent, but the general policy is that for new files the copyright should be the current year, but we don't update copyright on existing files.